app/models/piece.rb

```diff
@@ -36,7 +44,11 @@ def captured?
   def valid_move?(to_row, to_col)
     return false if move_nil?(to_row, to_col)
     return false if move_out_of_bounds?(to_row, to_col)
-    return false if move_destination_same_side?(to_row, to_col)
+    return false if move_destination_ally?(to_row, to_col)
+
+    # until have Knight::move_legal?
+    return true if type == "Knight"
```

Comment:
It's probably okay there for now, since it'll be killed in the knight PR no matter where it lives.  Moving it to `knight.rb` would probably get rid of the Rubocop error though....